### PR TITLE
fix: empty label exception should be labelCanNotBeNull()

### PR DESCRIPTION
**Pull request by seagle-yuan**
fix sub-issue 8 #2058

### CODE REVIEW

File: hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java [hunk 1]
```diff
@@ -430,13 +430,15 @@ public static final ElementKeys classifyKeys(Object... keyValues) {
...
-            if (val == null) {
-                throw Property.Exceptions.propertyDoesNotExist();
-            }
 
             if (key.equals(T.id)) {
                 elemKeys.id = val;
             } else if (key.equals(T.label)) {
+
+                if(val==null){
+                    throw Element.Exceptions.labelCanNotBeNull();
```
Comment (javeme) [inline hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java:439]: prefer to keep the `throw Property.Exceptions.propertyDoesNotExist();` and throw labelCanNotBeNull() if  `key.equals(T.label)`.

File: hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java [hunk 2]
```diff
@@ -430,13 +430,15 @@ public static final ElementKeys classifyKeys(Object... keyValues) {
...
             }
-            if (val == null) {
-                throw Property.Exceptions.propertyDoesNotExist();
-            }
 
             if (key.equals(T.id)) {
                 elemKeys.id = val;
             } else if (key.equals(T.label)) {
+
+                if(val==null){
```
Comment (javeme) [inline hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java:438]: expect a space before "(" and "{"

File: hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java [hunk 3]
```diff
@@ -55,6 +56,8 @@
 import org.apache.hugegraph.util.E;
 import org.apache.hugegraph.util.InsertionOrderUtil;
 
+import com.alipay.remoting.util.StringUtils;
```
Comment (javeme) [inline hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java:59]: useless import?

File: hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java [hunk 4]
```diff
@@ -430,15 +433,19 @@ public static final ElementKeys classifyKeys(Object... keyValues) {
...
                     throw Element.Exceptions.labelCanNotBeNull();
+                } else if (key.equals(T.label)) {
+                    throw Element.Exceptions.labelCanNotBeEmpty();
                 }
                 throw Property.Exceptions.propertyDoesNotExist();
             }
 
             if (key.equals(T.id)) {
                 elemKeys.id = val;
+
```
Comment (javeme) [inline hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java:448]: can remove the blank line

**Comment by javeme** [comment]
ci error
```java
Error:  Failures: 
Error:    AuthTest.testDeleteBelong:867 Bad exception type java.lang.IllegalStateException(expected java.lang.IllegalArgumentException)
Error:    AuthTest.testGetBelong:795 Bad exception type java.lang.IllegalStateException(expected java.lang.IllegalArgumentException)
Error:    VertexCoreTest.testAddVertexWithPrimaryValuesEmpty:841 Bad exception type java.lang.IllegalStateException(expected java.lang.IllegalArgumentException)
Error:  Errors: 
Error:    EdgeCoreTest.testQueryEdgeByPropertyWithEmptyString:6931 » IllegalState The pr...
Error:    VertexCoreTest.testAddVerticesWithUniqueIndexForNullableProperties:5946 » IllegalState
Error:    VertexCoreTest.testQueryVertexByPropertyWithEmptyString:6438 » IllegalState Th...
[INFO] 
Error:  Tests run: 731, Failures: 3, Errors: 3, Skipped: 81
```

File: .gitignore [hunk 5]
```diff
@@ -7,6 +7,8 @@ upload-files/
 demo*
 gen-java
 *.class
+apache-hugegraph-incubating-1.0.0/
+.github/
```
Comment (imbajin) [inline .gitignore:11]: we shouldn't ignore it

File: .gitignore [hunk 6]
```diff
@@ -7,6 +7,8 @@ upload-files/
 demo*
 gen-java
 *.class
+apache-hugegraph-incubating-1.0.0/
```
Comment (imbajin) [inline .gitignore:10]: it's redundent with line 61
image

**Review by imbajin — APPROVED** [review]

**Review by coderzc — APPROVED** [review]

**Pull request merged**